src/sentry/api/serializers/models/team.py

```diff
@@ -187,20 +197,43 @@ def serialize(
         return d
 
 
+def get_scim_teams_members(
+    team_list: Sequence[Team],
+) -> MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]]:
+    members = RangeQuerySetWrapper(
+        OrganizationMember.objects.filter(teams__in=team_list).select_related("user"),
+        limit=10000,
+    )
+    member_map: MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]] = defaultdict(list)
+    for member in members:
+        for team in member.teams.all():
```

Comment:
should i use a `prefetch_related` in the above query for teams? from what i read in the docs it doesn't seem like it would be that big of an improvement since its doing a separate lookup for each relationship

Reply:
i'm also wondering if there is an optimization to be had using `values` along with an annotation to get the user.email instead of calling member.get_email(), but leaving it as is for now.

Reply:
went ahead and added the prefetch_related, leaving off `value` use for now.